.redhat-ci.Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN dnf install -y \
         python-dbus \
         pylint \
         python-slip-dbus \
-        python-docker-py \
+        python2-docker \
```

Comment:
So another issue here is that F25 has Python v3.5, while rawhide has v3.6. Are we expecting incompatibilities here? Given that we'll be branching soon, maybe let's leave the redhat-ci testsuites for now and circle around to it when we add support for F26. I opened https://github.com/jlebon/redhat-ci/issues/28 to track that work.